### ISSUE TITLE
MaskModule

### ISSUE THREAD

**Issue by Branikas** [QUVA-Lab/e2cnn#5]
Hi all,
Thank you very much for your amazing framework!
I would like to ask a few things. Since I am trying to build an auto-encoder for image segmentation based on your framework, I was wondering about how and if I should use your mask module.
I have seen your example and read the documentation for the mask module. My question is, should I use it often? I have many blocks of sequential layers with different operations like convolutions, pooling, upsampling etc.
Do you think it's necessary to add a lot of them to maintain rotation equivariance?

Thank you.

**Comment by Branikas** [comment]
Another question, sorry for asking this separately. 
Have you tried storing and loading a model?
I want to store the model for each epochs and based on the validation round to choose which one I load.
However, when I tried storing the model with  torch.save(model.state_dict(), PATH) and loading with  model.load_state_dict(torch.load(PATH)) I get a big error log referring to the filters and the biases of different modules missing key values from the dictionary where they are stored. Any ideas?
Once again, thank you very much.
